ru/lessons/basics/mix.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: page
+title: Mix
+category: basics
+order: 9
+lang: ru
+---
+
+До того как мы начнем углубляться в специфику Elixir, сначала нужно изучить mix. Если вы знакомы с Ruby, то этот инструмент очень похож на объединение Bundler, RubyGems и Rake. Это очень важная часть любого Elixir проекта и в этом уроке мы собираемся рассмотреть некоторые его возможности. Для получения полного списка возможностей в mix есть команда `mix help`.
```

Comment:
In my opinion, `объединение Bundler, RubyGems и Rake` sounds a little bulky and unnaturally. I'd suggest `Bundler, RubyGems и Rake, вместе взятые`.
I'd use `ключевая` instead of `очень важная` and `откроем для себя` instead of `собираемся рассмотреть`.
I also guess that `лишь` could be used before `некоторые его возможности` because there is  `just a few` expression in the original sentence.
A comma should be used before `в этом уроке` (https://ru.wikipedia.org/wiki/Сложносочинённое_предложение).